src/testkit/scala/tools/testkit/AssertUtil.scala

```diff
@@ -201,9 +201,41 @@ object AssertUtil {
 
   /** Like Await.ready but return false on timeout, true on completion, throw InterruptedException. */
   def readyOrNot(awaitable: Awaitable[_]): Boolean = Try(Await.ready(awaitable, TestDuration.Standard)).isSuccess
+
+  /** Syntax to clarify which is the expected value. */
+  def assertEqualTo[A](expected: A)(actual: A): Unit = assertEquals(expected, actual)
```

Comment:
Is it also part of the intent to make them not LUB?  And/or is it so the second one can be a block?